gobblin-core/src/main/resources/mysql/mysql.properties

```diff
@@ -16,7 +16,7 @@
 #
 
 # Source properties - source class to extract data from Mysql Source
-source.class=gobblin.source.extractor.extract.jdbc.MysqlSource
+source.class=gobblin.source.jdbc.MysqlSource
```

Comment:
Changing the fully qualified name of the class is dangerous, as all deployed applications will need to be updated. Is this needed?